Makefile.am

```
@@ -100,6 +100,7 @@ SUBDIRS += \
     streaming \
     web \
     claim \
+	aclk \
```

Please use spaces here instead of tabs

```suggestion
    aclk \
```